requirements/edx/github.txt

```diff
@@ -93,7 +93,7 @@ git+https://github.com/edx/edx-milestones.git@v0.1.11#egg=edx-milestones==0.1.11
 git+https://github.com/edx/xblock-utils.git@v1.0.5#egg=xblock-utils==1.0.5
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@138e6fa0bf3a2013e904a085b9fed77dab7f3f21#egg=xblock-google-drive
 git+https://github.com/edx/edx-user-state-client.git@1.0.2#egg=edx-user-state-client==1.0.2
-git+https://github.com/edx/xblock-lti-consumer.git@v1.1.6#egg=lti_consumer-xblock==1.1.6
+git+https://github.com/proversity-org/xblock-lti-consumer.git@4992c95bd188942059cce7d9d704666342bbe3bd#egg=lti_consumer-xblock==1.1.6
```

Comment:
Why from commit? What about the versioning? @andrey-canon

Reply:
We do that when we have to update old version and the next version already exists @Squirrel18

Reply:
Got it. @andrey-canon